### PR TITLE
callout for turbo mode

### PR DESCRIPTION
**Pull request by s-santillan**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [x] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [x] Redirects are added if the PR changes page URLs
- [x] If you have changed any header tag links (doc/#this-kind-of-anchor), update all instances of that link

Minor update to mention turbo mode. We emphasize our gentle push for its use through a tip box (in green) and mention it is not on by default ([per this slack thread](https://semgrepinc.slack.com/archives/C04RL2BBL2V/p1684351574049819)) and is in beta. I did test that it runs on both simple and advanced mode.

Snag_9c76b0

### CODE REVIEW

**Comment by s-santillan** [comment]
Have added T's suggestion in.

File: src/components/procedure/_enable-turbo-mode.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+:::tip Try Turbo mode (beta)
+Semgrep can automatically scan for findings using the Semgrep OSS Engine, on any change in the Rule or Sample Code panes.
```
Comment (adamkvitek) [inline src/components/procedure/_enable-turbo-mode.md:2]: I was just checking [the pane definition](https://developers.google.com/style/ui-elements#term-windows) also in [Stack Exchange](https://ux.stackexchange.com/questions/124588/whats-the-difference-between-a-panel-and-pane). Looks correct - as pane is a section of a window. So the Editor is a page, with a window that takes roughly four-fifths of its size, with a left sample code pane and a right test code pane. Feel free to resolve this comment, and if you want send me more details so I can be sure we shouldn't use a "panel". :D

**Review by adamkvitek — APPROVED** [review]
@s-santillan LGTM. 🎉

**Comment by s-santillan** [comment]
Thank you all. Will wait for Turbo mode to be available in Playground before merging.

**Pull request merged**